addon/chart-data-updater.js

```diff
@@ -36,8 +36,14 @@ export default Ember.Object.extend({
           item = item || 0;
           if(typeof chartDataset.bars !== 'undefined') {
             chartDataset.bars[j].value = item;
+
+            // Update tooltip labels
+            chartDataset.bars[j].label = labels[j];
           } else {
             chartDataset.points[j].value = item;
+
+            // Update tooltip labels
+            chartDataset.points[j].label = labels[j];
```

Comment:
Because we need to either reference `bars` or `points` off of `chartDataset` depending on the type of chart.  Therefore these lines are not duplicates of each other but distinct.